libr/anal/p/anal_x86_cs.c

```diff
@@ -145,7 +145,7 @@ static void opex(RStrBuf *buf, cs_insn *insn, int mode) {
 				r_strbuf_appendf (buf, ",\"index\":\"%s\"", cs_reg_name (handle, op->mem.index));
 			}
 			r_strbuf_appendf (buf, ",\"scale\":%d", op->mem.scale);
-			r_strbuf_appendf (buf, ",\"disp\":%"PFMT64u"", op->mem.disp);
+			r_strbuf_appendf (buf, ",\"disp\":%lu", op->mem.disp);
```

Comment:
probably this should be
```suggestion
			r_strbuf_appendf (buf, ",\"disp\":%"PFMT64u, (ut64)op->mem.disp);
```

Reply:
I need to change also all the others accordingly.

Reply:
looks wrong to me, this variable is signed

include/capstone/x86.h: int64_t disp;

Reply:
Good point. PFMT64d then.